packages/services/api/src/modules/schema/providers/schema-publisher.ts

```diff
@@ -720,22 +720,26 @@ export class SchemaPublisher {
           })) ?? null,
       });
     } else if (checkResult.conclusion === SchemaCheckConclusion.Skip) {
-      if (!comparedVersion || !comparedSchemaVersion) {
+      // SchemaCheckConclusion.Skip is returned by the checksum check
+      // when the schema is the same as the previous one.
+      // To perform the check we take the previous version (not necessarily composable)
+      // and that's why we ensure that `latestVersion` and `latestSchemaVersion` are available.
+      if (!latestVersion || !latestSchemaVersion) {
```

Comment:
Okay I think I now understand that in case there is a skip, the compared version is always the latest version.